src/component/mxgraph/BpmnMxGraph.ts

```diff
@@ -36,7 +36,16 @@ export class BpmnMxGraph extends mxGraph {
     return scale;
   }
 
+  // override fit to set initial cumulativeZoomFactor
+  zoomActual(): void {
+    super.zoomActual();
+    this.cumulativeZoomFactor = this.view.scale;
+  }
+
   public customFit(fitOptions: FitOptions): void {
+    // To fix weird behaviour when the diagram is already fit
```

Comment:
:warning: let's add ref to the issue explaining this
```suggestion
    // TODO avoid extra zoom/fit reset
    // see https://github.com/process-analytics/bpmn-visualization-js/issues/888
```